prod/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.7"
+services:
+  mongo:
+    image: mongo
+    restart: always
+    init: true
+    ports:
+      - "27017:27017"
+    environment:
+      MONGO_INITDB_DATABASE: "covid19"
+  curator:
```

Comment:
Do we want to use this approach for the dev stack, too?

Reply:
sorry what approach? Using this image for the curator service you mean?
We could but it would be wasteful as we'd still need to build the UI separately (ending up building it twice) and start it with the react tool dev scripts.
Unless I misunderstood your question? sorry please clarify if so.

Reply:
Yeah, using the single image. Plus side is that our dev testing would more closely reflect the prod configuration. Why do we need to still build the UI separately in the dev compose set up?

Reply:
You certainly don't notice that on your überpowerful machine but for me it takes a few minutes to do an initial build (incremental builds are fine) ;-)
Building the images separately in dev allows us to not produce an (unused) optimized build of the UI.
BUT it also has the nice effect of allowing us to test the prod UI in with the dev curator service without having to run prod/run_stack.sh ...
So yeah, meh, I can deal with the longer initial build times I think.
Done.